### ISSUE TITLE
[styleguide] Panel buttons don't use .btn

### ISSUE THREAD

**Issue by jadekler** [pivotal-cf/pivotal-ui#489]
Current documentation in styleguide for panels:

```
<Panel className="bg-neutral-10" header={<h2>Custom Title</h2>} actions={<div><button>Go</button><button>Stop</button></div>}>
  Base Panel with custom header and actions
</Panel>
```

`<button>` is not a style we define in [button.scss](https://github.com/pivotal-cf/pivotal-ui/blob/40af1c0bb4ba1f53c2c36bcaa32ad13f641f64f1/library/src/pivotal-ui/components/buttons/buttons.scss), so I'm unsure how this is being rendered. However, I believe this should be `<button className="btn">`.

**Comment by cwang-pivotal** [comment]
This has been fixed in 8.0; I believe we're not publishing styleguide until the release has hardened a bit more

**Comment by charleshansen** [comment]
fixed
